blank-canvas/inc/wpcom.php

```diff
@@ -23,7 +23,7 @@ function blank_canvas_register_wpcom_block_patterns() {
 		
 		if ( function_exists( 'register_block_pattern' ) ) {
 
-			/* This pattern is WP.com-Only Beacuse it relies on the Layout Grid block. */
+			/* This pattern is WP.com-only because it relies on the Layout Grid block. */
```

Comment:
@kjellr is there some additional function_exists() to check if Jetpack Layout Grid is present that could be added here? Or are there any WordPress plugin or block check functions that could be checked here for throwing notices in /wp-admin?

Reply:
I did look into that briefly, but I didn't find anything obvious. There don't seem to be any functions I could check for in [the plugin source](https://plugins.svn.wordpress.org/layout-grid/trunk/). 

That could use another set of eyes though if someone else has a moment.

Reply:
@onemaggie and I took a look at getting that working on Friday, but we didn't make any progress. 

Since the current solution (load these only on WP.com) technically works ok, I suggest we merge and follow up with a better fix later on. 

The only effected parties here are folks who use this theme outside of WP.com and also have the Layout Grid plugin installed. And even in that case, they'll currently just not see this pattern, which seems fine. 

Does that work for folks? 

If so, would someone mind giving this a final review?